dask/bag/core.py

```diff
@@ -880,19 +880,44 @@ def combine2(acc, x):
                                 list(dsk.keys())))}
         return type(self)(merge(self.dask, dsk, dsk2), b, 1)
 
-    def take(self, k, compute=True):
+    def take(self, k, npartitions=1, compute=True):
         """ Take the first k elements
 
         Evaluates by default, use ``compute=False`` to avoid computation.
-        Only takes from the first partition
+        Only takes elements from the first ``npartitions``, may result in fewer
+        than k elements. default ``npartitions`` is 1, use -1 for all
+        available partitions.
+
+        a warning will be raised if less than ``k`` elements are returned.
```

Comment:
Perhaps:

```python
""" Take the first k elements.

Parameters
----------
k : int
    The number of elements to return.
npartitions : int, optional
    Elements are only taken from the first ``npartitions``, with a default of 1.
    If there are fewer than ``k`` elements in the first ``npartitions`` a warning
    will be raised and any found elements returned. Pass -1 to use all partitions.
compute : bool, optional
    Whether to compute the result, default is True.

...
"""
```

(note, I didn't check line wrapping is < 80 chars, may need to fix).